src/server/scripts/Northrend/Ulduar/Ulduar/boss_xt002.cpp

```diff
@@ -282,8 +285,7 @@ class boss_xt002 : public CreatureScript
                         case EVENT_SEARING_LIGHT:
                             if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 0))
                                 DoCast(target, SPELL_SEARING_LIGHT);
-
-                            events.ScheduleEvent(EVENT_SEARING_LIGHT, TIMER_SEARING_LIGHT);
+                            events.ScheduleEvent(EVENT_SEARING_LIGHT, TIMER_SEARING_LIGHT * 2);
```

Comment:
The boss alternates between casting Gravity Bomb and Searing light, I changed the original time of searing light to be half what it was (as some others suggested) and put the *2 in so every time it was rescheduled after being cast it would be cast at the same time as gravity bomb